evm/gateway.js

```diff
@@ -508,6 +511,9 @@ if (require.main === module) {
 
     program.addOption(new Option('--payload <payload>', 'gmp payload'));
     program.addOption(new Option('--commandID <commandID>', 'execute command ID'));
+    program.addOption(new Option('--messageId <messageId>', 'GMP call message ID'));
+    program.addOption(new Option('--sourceChain <sourceChain>', 'GMP source chain'));
+    program.addOption(new Option('--sourceAddress <sourceAddress>', 'GMP source address'));
     program.addOption(new Option('--destination <destination>', 'GMP destination address'));
     program.addOption(new Option('--destinationChain <destinationChain>', 'GMP destination chain'));
```

Comment:
why not merge all of this into a single flag that receives a json object?

Reply:
this will be cleaned up to use nested CLI cmds (similar to sui/stellar scripts). Using JSON input is a bit worse UX wise when using manually